### PR TITLE
Add databroker

### PR DESCRIPTION
**Pull request by argerus**
Signed-off-by: John Argérus <fixed-term.john.argerus@se.bosch.com>

### CODE REVIEW

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
Hi @argerus  , can you also modify the main project Readme table with a small row referencing data broker? 

For example, something like this

| License | Component      | Description |
|--|--|--|
| [License](https://opensource.org/licenses/Apache-2.0) | [kuksa-databroker](kuksa_databroker) | Efficient in-vehicle signal broker written in RUST    |

We probably need to get more wordy there in the near future, but for now let's at least have a complete TOC 😄

**Comment by argerus** [comment]
> Hi @argerus , can you also modify the main project Readme table

done

**Review by SebastianSchildt — COMMENTED** [review]
Looking good, just two minor comments/questions ^^

File: kuksa_databroker/README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+## Relation to the COVESA Vehicle Signal Specification (VSS)
+
+The data broker is designed to support data entries and branches as defined by [VSS](https://covesa.github.io/vehicle_signal_specification/).
+
+In order to generate metadata from a VSS specification that can be loaded by the data broker, it's possible to use the `vspec2json.py` tool
+that's available in the [vss-tools](https://github.com/COVESA/vss-tools) repository. E.g.
+
+```shell
+./vss-tools/vspec2json.py -I spec -i :uuid.txt spec/VehicleSignalSpecification.vspec vss.json
```
[the remark this answers is not in the document]
Comment (argerus) [reply kuksa_databroker/README.md]: > Outdated?

Yes, fixed.

**Review by SebastianSchildt — APPROVED** [review]
lgtm

**Pull request merged**